app/js/lib/server/network.js

```diff
@@ -124,16 +125,13 @@ export default class Network extends EventDispatcher {
     return fetch(url, req)
       .then((res) => {
         if (!res.ok) {
-          throw new TypeError(
+          throw new HttpError(
+            res.status,
             `The response returned a ${res.status} HTTP status code.`
           );
         }
 
         return res;
-      })
-      .catch((error) => {
-        console.error('Error occurred while fetching content: ', error);
-        throw error;
```

Comment:
useless; the devtools are showing it we don't catch it anyway, so because we don't do anything useful here, better just remove it.